### PR TITLE
Use explicit volumes instead of implicit volumes

### PR DESCRIPTION
**Pull request by pchico83**

### CODE REVIEW

**Review by rberrelleza — APPROVED** [review]

File: okteto-stack.yml [hunk 1]
```diff
@@ -1,22 +1,24 @@
 name: voting-app
+
+endpoints:
```
Comment (rberrelleza) [inline okteto-stack.yml]: if this is a getting started example, wouldn't it be easier to go with `public: true`?  I think the `endpoints` key should be reserved for the scenarios where it provides clear value (e.g. specify a non-standard route)

Comment (pchico83) [reply okteto-stack.yml]: The problem is that you need to explain what `public` is and how it works.
`endpoints` (level 7 proxy) is already a concept for developers, it is less magic to explain

Comment (rberrelleza) [reply okteto-stack.yml]: Endpoints and a proxy are two different concepts. I think it's going to be hard for someone to read 'endpoint' and realize that it is the equivalent of doing a proxy to one of their services. I think that making a service `public` is a lot easier to grasp for someone who's getting started with stacks  (that is the whole reason why that feature exists in the first place no?).

Comment (pchico83) [reply okteto-stack.yml]: I have kept it with `public ` for now.
But I keep getting the same feedback from people that having several ways to do the same thing is very confusing

Comment (rberrelleza) [reply okteto-stack.yml]: 100% agree. if we want to only support endpoints that's ok too, we just need to make sure is well documented.

**Pull request merged**